pkg/prometheus/promcfg.go

```diff
@@ -449,6 +476,9 @@ func (cg *ConfigGenerator) generatePodMonitorConfig(
 	enforcedNamespaceLabel string,
 	enforcedSampleLimit *uint64,
 	enforcedTargetLimit *uint64,
+	labelLimit *uint64,
+	labelNameLengthLimit *uint64,
+	labelValueLengthLimit *uint64,
```

Comment:
Since these come from the Prometheus level let's prefix the variable name by `enforced` as in `enforcedSampleLimit`